module/VuFind/src/VuFind/Form.php

```diff
@@ -0,0 +1,610 @@
+<?php
+/**
+ * Configurable form.
+ *
+ * PHP version 7
+ *
+ * Copyright (C) The National Library of Finland 2018.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ *
+ * @category VuFind
+ * @package  Form
+ * @author   Samuli Sillanpää <samuli.sillanpaa@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:controllers Wiki
+ */
+namespace VuFind;
+
+use VuFind\Config\YamlReader;
+use Zend\InputFilter\InputFilter;
+use Zend\Validator\EmailAddress;
+use Zend\Validator\NotEmpty;
+
+/**
+ * Configurable form.
+ *
+ * @category VuFind
+ * @package  Form
+ * @author   Samuli Sillanpää <samuli.sillanpaa@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:controllers Wiki
+ */
+class Form extends \Zend\Form\Form
+{
+    /**
+     * Input filter
+     *
+     * @var InputFilter
+     */
+    protected $inputFilter;
+
+    /**
+     * Validation messages
+     *
+     * @var array
+     */
+    protected $messages;
+
+    /**
+     * Default form config (from config.ini > Feedback)
+     *
+     * @var array
+     */
+    protected $defaultConfig;
```

Comment:
Unless I'm missing something, this property is no longer being used -- I think it can be removed from here, the constructor, and the factory.

Reply:
Oops, this should be $defaultFormConfig which is used.